packages/desktop-client/src/components/modals/MergeUnusedPayees.js

```diff
@@ -118,7 +118,7 @@ export default function MergeUnusedPayees({
                     }}
                   >
                     {payees.map(p => (
-                      <li>
+                      <li key={p.id}>
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks!